.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
[nitpick] I highly recommend deleting this file from your pull request.
```suggestion
    "git.ignoreLimitWarning": true
```

Reply:
Noted with thanks.